ui/apps/platform/src/utils/getTableUIState.ts

```diff
@@ -56,11 +50,7 @@ export function getTableUIState<DataType>({
         return { type: 'ERROR', error };
     }
 
-    if (isLoading && isPolling && data) {
-        return { type: 'POLLING', data };
-    }
-
-    if (isLoading) {
+    if (isLoading && !isPolling) {
```

Comment:
the table ui state was not handling polling correctly. Here's an example of what it did before:

https://github.com/user-attachments/assets/a2b8abdb-2d60-4eb3-9920-97e002c9dffa

We won't show the spinner when polling. Instead, we will keep the current state. We need to consider how we deal with polling. It's not in an ideal state yet.